app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
```

Comment:
- [ ] Good job on creating the models and Setting up associations between them, Kindly also create the requested custom methods in all models as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project_v1_1.md#:~:text=Create%20custom%20methods,likes%20counter%20for%20a%20post.).